styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -251,49 +253,53 @@ private void triggerBackfills(Collection<InstanceState> activeStates) {
             counting()));
 
     backfills.forEach(backfill -> {
-      final BackfillBuilder builder = backfill.builder();
 
```

Comment:
a general comment. this method is getting too long to follow. we will need a refactoring later. but now it is ok.